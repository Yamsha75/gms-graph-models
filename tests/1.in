1 348
1 385
1 775
1 140
1 821
1 515
1 423
1 114
1 783
1 179
1 985
1 486
1 405
1 709
1 299
1 668
1 440
1 121
1 930
1 971
1 72
1 337
1 582
1 58
1 602
1 302
1 741
1 465
1 128
1 834
1 65
1 973
1 974
1 894
1 804
1 816
1 875
1 425
1 713
1 304
1 616
1 28
1 117
1 991
1 146
1 539
1 79
1 511
1 785
1 428
1 85
1 523
1 69
1 554
1 221
1 51
1 913
1 273
1 806
1 929
1 717
1 318
1 586
1 667
1 931
1 33
1 672
1 208
1 468
1 107
1 521
1 901
1 335
1 90
1 358
1 546
1 436
1 105
1 581
1 817
1 900
1 561
1 272
1 835
1 454
1 811
1 10
1 213
1 827
1 820
1 70
1 943
1 392
1 32
1 474
1 316
1 216
1 252
1 421
1 846
