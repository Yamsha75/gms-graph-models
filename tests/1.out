177472
218310
916826
26722
1032685
395836
264847
17304
936956
44748
1495250
351419
242362
765167
129804
676400
287190
19662
1331554
1452715
6585
166214
509954
4145
546291
132438
837171
321465
22073
1066169
5259
1458790
1461846
1227788
989895
1020288
1175252
267326
773718
134240
572356
832
18268
1513896
29359
434669
7988
389124
942372
271113
9337
408498
5975
459766
69275
3136
1283126
107515
994594
1328869
782492
147258
516822
674213
1334240
1220
685372
61281
325629
15226
405229
1247496
163935
10591
188099
446688
282072
14621
508325
1023071
1244796
472815
106703
1068565
306067
1006786
76
64282
1047600
1030229
6187
1368616
226736
1132
334152
145327
66161
90998
262488
1097061
